tests/core/full_node/test_mempool.py

```diff
@@ -1406,7 +1422,7 @@ async def test_stealing_fee(self, two_nodes):
         )
 
         full_node_1, full_node_2, server_1, server_2 = two_nodes
-        peer = await connect_and_get_peer(server_1, server_2)
+        peer = await connect_and_get_peer(server_1, server_2, bt.config["self_hostname"])
```

Comment:
Should this use the `self_hostname` fixture?

Reply:
Looks like it would apply to 9 occurrences in this file.

Reply:
I just did this for convenience becase we already have bt. `self_hostname` depends on `bt`, so it is guaranteed to be the same